lib/Qandidate/Toggle/Serializer/ContextSerializer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Qandidate\Toggle\Serializer;
+
+use Qandidate\Toggle\Context;
+
+class ContextSerializer
+{
+    /**
+     * @param array $data
+     *
+     * @return Context
+     */
+    public function deserialize(array $data)
+    {
+        $this->assertHasKey('context', $data);
```

Comment:
Well, the idea was to deserialize the whole request when asking for a toggle, and the request contains two entries : the name of the toggle and the context.

To make things right, using the pattern already in place for the toggle adding request for instance, I'm guessing we should have another serializer that would have the responsibility to check for a 'name' and 'context' entries and then call the `ContextSerializer` when needed, but it kind of seems a bit overkill to me.

What do you think ? Thank you for your feedback BTW :)